specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -492,6 +635,203 @@
           }
         }
       }
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Active Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nGet all government-issued active storms.",
```

Comment:
Change:  
Get all government-issued active storms.

To:  
Get all government-issued active tropical storms. Information about the tropical storms include, government ID, basin ID, year of origin, name and if it is subtropical.

Reply:
Updated